docs/m2c_basics.md

```diff
@@ -0,0 +1,16 @@
+# M2C Basics
+[m2c](https://simonsoftware.se/other/m2c.html) is an alternative decompilation tool that can be used to help reverse engineer functions. While it is a more limited web-based tool it can generate alternative output to tools such as Ghidra which can be helpful as it sometimes has more accurate results.
```

Comment:
>  While it is a more limited web-based tool...

This isn't true, there's a local m2c script that you can use, I just use the online tool since it's more comfortable to use,  though the web version is limited to a certain amount of functions, while the local one can handle big files
[Here](https://github.com/matt-kempster/m2c)